src/mainboard/pcengines/apu2/Kconfig

```diff
@@ -33,7 +33,7 @@ config BOARD_SPECIFIC_OPTIONS # dummy
 	select GENERIC_SPD_BIN
 	select SPI_FLASH
 	select SPI_FLASH_WINBOND
-	select TPM
+	select TPM1
```

Comment:
@miczyg1 are we sure this is correct?

Reply:
@miczyg1 https://github.com/pcengines/coreboot/blob/014d812ebdea72c7046de1c2e2022939439e999a/src/security/tpm/Kconfig#L22

Reply:
@pietrushnic question is why we had TPM selected in the first place. If we want to keep things as it were before, we should keep it. Either way it is selected by two options below:
```
MAINBOARD_HAS_LPC_TPM
MAINBOARD_HAS_TPM1
```
This select does not affect anything:
https://github.com/pcengines/coreboot/blob/014d812ebdea72c7046de1c2e2022939439e999a/src/security/tpm/Kconfig#L23

Reply:
@miczyg1 yes, but our module is SLB9665 TPM2.0

Reply:
@pietrushnic Yes and probably we Will need to push patches to enable it. For now MAINBOARD_HAS_TPM1 has been added in mainline due to some bigger change to TPMs code. If we want to support mentioned module, we have to verify firstly what we need to enable in the code.

Reply:
@miczyg1 agree. This is on my TODO right after IOMMU.